pylint/config.py

```diff
@@ -526,6 +527,28 @@ def parse(self, to_parse, config):
         pass
 
 
+def make_commenter(description):
+    """Make a function to add comments from re.sub."""
+    def make_comment(match):
+        """Make the replacement string including commented desciption."""
+        comment = '\n'.join(
+            '# {}'.format(line) for line in textwrap.wrap(description))
```

Comment:
This will wrap to 72 chars (70 default for wrap, plus two for `'# '`), not 79 like `generate_config` did. That's personal preference, I think 72 is nicer for prose (as per PEP 8 for doc strings and comments).